akka-http/src/main/scala/akka/http/scaladsl/server/RequestContext.scala

```diff
@@ -43,6 +44,11 @@ trait RequestContext {
   def settings: RoutingSettings
 
   /**
+   * The default ParserSettings to be used for configuring directives.
+   */
+  def parserSettings: ParserSettings
```

Comment:
Is this used somewhere? In any case, it would make sense to keep the RequestContext lean. If we need it here at all we should maybe move it into RoutingSettings? Better would be to remove it if it isn't used, though :)